### ISSUE TITLE
Why is the result of each pre-training model unstable?

### ISSUE THREAD

**Issue by CoisiniYrq** [tyhuang0428/CLIP2Point#7]
Dear sir,
I ran the "pretraining.py" three times, and use models to "zeroshot.py", the results are 50%  50.44% 49.5% respectively, it is not stable! Do you know what caused it?
Thanks for your attention.

**Comment by tyhuang0428** [comment]
The ShapeRender dataloader in pre-training randomly selects viewpoints. The ModelNet40Align dataloader in zero-shot randomly sample points. It is normal that the results are slightly different. You may set a random seed.

**Comment by CoisiniYrq** [comment]
Thank you for your answer, I will try this method.
